agent/util-scripts/pbench-register-tool-set

```diff
@@ -129,6 +129,70 @@ if [[ -z ${the_tool_set} ]]; then
 	exit 1
 fi
 
+
+# This function checks a --remotes file argument  and if everything
```

Comment:
Before we duplicate this code, can we consider PR https://github.com/ndokos/pbench/pull/13?

Reply:
Done.

Reply:
Yes, I pulled it.